LoRaEngine/LoraKeysManagerFacade/DeviceFeedConstants.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
```

Comment:
Enable nullable context in new source files:

```suggestion
#nulleable enable

namespace LoraKeysManagerFacade
```

Reply:
Hi @Atif, 

Thank you for this suggestion. 
I didn't know this annotation. could you explain me what does it provide and when its interresting to use ?